Makefile

```diff
@@ -112,17 +112,16 @@ go-generate:
 	@ # we use this in pkg/controller/common/license
 	go generate -tags='$(GO_TAGS)' ./pkg/... ./cmd/...
 
-generate-crds-v1: go-generate controller-gen
```

Comment:
Why is go-generate being removed? Was this just unneeded at this point?

Reply:
It should only be necessary before we build the actual go binaries for the operator. The only thing it does is embed the license public key into source code.

Reply:
Yes, it's useless!

We generate two things: the pub key (`./pkg/controller/common/license/zz_generated.pubkey.go`) and the boilerplate to deep copy (`zz_generated.deepcopy.go`).

```
> find . -name "zz*"
./pkg/apis/maps/v1alpha1/zz_generated.deepcopy.go
./pkg/apis/enterprisesearch/v1/zz_generated.deepcopy.go
./pkg/apis/enterprisesearch/v1beta1/zz_generated.deepcopy.go
./pkg/apis/apm/v1/zz_generated.deepcopy.go
./pkg/apis/apm/v1beta1/zz_generated.deepcopy.go
./pkg/apis/kibana/v1/zz_generated.deepcopy.go
./pkg/apis/kibana/v1beta1/zz_generated.deepcopy.go
./pkg/apis/stackconfigpolicy/v1alpha1/zz_generated.deepcopy.go
./pkg/apis/logstash/v1alpha1/zz_generated.deepcopy.go
./pkg/apis/beat/v1beta1/zz_generated.deepcopy.go
./pkg/apis/agent/v1alpha1/zz_generated.deepcopy.go
./pkg/apis/common/v1/zz_generated.deepcopy.go
./pkg/apis/common/v1beta1/zz_generated.deepcopy.go
./pkg/apis/common/v1alpha1/zz_generated.deepcopy.go
./pkg/apis/autoscaling/v1alpha1/zz_generated.deepcopy.go
./pkg/apis/elasticsearch/v1/zz_generated.deepcopy.go
./pkg/apis/elasticsearch/v1beta1/zz_generated.deepcopy.go
./pkg/controller/common/license/zz_generated.pubkey.go
```
```
# cleanup
> find . -name "zz*" | xargs -n1 rm -f
# check
> find . -name "zz*"
# generate!
> make generate-manifests
# -- generate  webhook manifest
# -- generate  crd bases manifests
# -- kustomize crd manifests
# -- generate  crds manifest
# -- generate  operator manifest
```

Reply:
Thanks to both of you for this explanation. This helped clear up some things